tools/CMakeLists.txt

```diff
@@ -26,3 +33,35 @@ if (SEA_DSA_STATIC_EXE)
   set_target_properties (seadsa PROPERTIES LINK_SEARCH_START_STATIC ON)
   set_target_properties (seadsa PROPERTIES LINK_SEARCH_END_STATIC ON)
 endif()
+
+file(
+  MAKE_DIRECTORY
+  ${SPEC_DIR}
+)
+
+add_custom_command(
+  OUTPUT ${SPEC_DIR}/libc.spec.c 
+  DEPENDS libcSpecGen 
+  COMMAND libcSpecGen
+  COMMAND clang-format -i ${SPEC_DIR}/libc.spec.c
+)
+
+add_library(LibcSpecs OBJECT ${SPEC_DIR}/libc.spec.c)
+set_target_properties(LibcSpecs PROPERTIES 
```

Comment:
you expect that everything is compiled with clang. This is not always the case. Perhaps a warning or at least a comment in the cmake file should be given in case someone compiles with gcc instead